### DISCUSSION TITLE
Grafana for Windows - HTTPS server configuration

### DISCUSSION THREAD

**Discussion by 3BK** [grafana/grafana#68656] [Platform]
I would like Grafana to read the custom.ini, read the SHAKE/SHA256/SHA1 digest of the HTTPS certificate in question, and use the HTTPS cert which is in the Windows Certificate Store.

https://community.grafana.com/t/grafana-https-server-inquiry/87834/1

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). I see there was a thread made in the community forum about this, so discussion can continue there. Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
